packages/ckeditor5-table/theme/table.css

```diff
@@ -42,6 +42,23 @@
 	}
 }
 
+/**
+ * Expanding the table to the full height of the parent container is necessary due to the fact that
+ * the tables are rendered inside a <figure> elements which are kinda buggy in terms of table height calculation.
+ * While setting `height: 100%` fixes the issue in the editing mode described here:
+ * https://github.com/ckeditor/ckeditor5/issues/6186
+ *
+ * it's causing another issue with the table height in the print preview mode here:
+ * https://github.com/cksource/ckeditor5-commercial/issues/6507
```

Comment:
Please do not link to the private repo. Anyone reading this code without access to the private would be unable to see it.

Reply:
@niegowski Done